frappe/integrations/doctype/google_settings/google_settings.py

```diff
@@ -2,11 +2,29 @@
 # Copyright (c) 2019, Frappe Technologies and contributors
 # For license information, please see license.txt
 
-# import frappe
+import frappe
 from frappe.model.document import Document
 
 class GoogleSettings(Document):
 	pass
 
 def get_auth_url():
-	return "https://www.googleapis.com/oauth2/v4/token"
\ No newline at end of file
+	return "https://www.googleapis.com/oauth2/v4/token"
+
+
+@frappe.whitelist(allow_guest=True)
+def get_file_picker_settings():
+	"""Return all the data FileUploader needs to start the Google Drive Picker."""
+	if frappe.session.user == 'Guest':
+		return {'enabled': False}
```

Comment:
Another nitpick, but I think Frappe's using Black now and is enforcing double-quotes for strings.